test/integration/tracks_test.go

```diff
@@ -0,0 +1,30 @@
+//go:build integration
+// +build integration
+
+package integration
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"strconv"
+	"testing"
+)
+
+func TestTracks(t *testing.T) {
+	var kind int = 1695506
+	ctx := context.Background()
+	t.Run("Get track information", func(t *testing.T) {
+		result, resp, err := client.Tracks().Get(ctx, kind)
+		assert.NoError(t, err)
+		assert.NotZero(t, result)
+		assert.Equal(t, http.StatusOK, resp.StatusCode)
+		assert.NotZero(t, result.Result)
+		assert.Equal(t, result.Result[0].ID, strconv.Itoa(kind))
```

Comment:
If `result.Result` is empty it will panic with `index out of range`. To avoid that you can use `require.NotZero()` above instead of `assert.NotZero()` from `github.com/stretchr/testify/require`. `require` interrupts test if it's failed

Reply:
P.S. Actually I suggest to use `require` everywhere as I like it now more than `assert`

Reply:
> P.S. Actually I suggest to use `require` everywhere as I like it now more than `assert`

Okay, I'll also change it

Reply:
@ndrewnee should I change `assert` to `require` in other files?

Reply:
Yeah you can change